src/types/locales.ts

```diff
@@ -17,7 +17,7 @@ export type SupportedLanguages =
   | 'nl'
 
 export type LocaleConfig = {
-  locale?: SupportedLanguages
+  locale: SupportedLanguages
```

Comment:
This is a breaking change though, right?

Reply:
I reverted it back to being optional.